src/dashboard/general/CustomTable/CustomTable.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  Table,
+  TableBody,
+  TableHead,
+  TableCell,
+  TableRow,
+  Paper,
+  Typography
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { createUseStyles } from 'react-jss';
+const useStyles = createUseStyles({
```

Comment:
styles in new file and import

Reply:
Saving this for later cleanup task mentioned above.